pkg/local-disk-manager/disk/manager/partition.go

```diff
@@ -29,4 +29,7 @@ type PartitionInfo struct {
 
 	// Filesystem
 	Filesystem string
+
+	// Path
+	Path string
```

Comment:
does this path represent the partition-path like `/dev/sda1`,`/dev/sda2`? 

i'm wondering if it's a mount-path, but I think the partition-path should be more meaningful.

Reply:
Yes,its format turly like `/dev/sda1`. It comes from the field `DEVNAME`  of `udevadm info` command.